lib/matplotlib/backend_bases.py

```diff
@@ -919,9 +919,15 @@ def set_antialiased(self, b):
         self._antialiased = int(bool(b))
 
     def set_capstyle(self, cs):
-        """Set the capstyle to be one of ('butt', 'round', 'projecting')."""
-        validate_capstyle(cs)
-        self._capstyle = cs
+        """
+        Set the `.CapStyle`.
+
+        Parameters
+        ----------
+        cs : `.CapStyle` or {'butt', 'round', 'projecting'}
+            How to draw end points of lines.
```

Comment:
Not a strong opinion, but this may read a bit better:

```suggestion
        Set how to draw end points of lines.

        Parameters
        ----------
        cs : `.CapStyle` or {'butt', 'round', 'projecting'}
```